spec/lucky/action_rendering_spec.cr

```diff
@@ -52,9 +52,11 @@ end
 describe Lucky::Action do
   describe "rendering HTML pages" do
     it "render assigns" do
-      body = Rendering::Index.new(build_context, params).call.body
+      response = Rendering::Index.new(build_context, params).call
+      body = response.body
```

Comment:
Instead of assigning to a variable, you could do `response.body.should...` on the line below

Reply:
👍